.github/workflows/ci.yml

```diff
@@ -0,0 +1,157 @@
+name: Build VV
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  INSTALL_PREFIX: "${{ github.workspace }}/install"
```

Comment:
As far as I understand, I think all install steps can be removed as the install directory is not used anywhere.